src/main.py

```diff
@@ -21,28 +25,85 @@
 
 from src.api.aws_translate_api import AWSTranslateAPI
 
+# Function to log progress
+class TqdmToLogger:
+    def __init__(self, logger, level=logging.INFO):
+        self.logger = logger
+        self.level = level
+        self.last_message = None
+
+    def write(self, message):
+        message = message.strip()
+        if message != self.last_message:
+            self.last_message = message
+            self.logger.log(self.level, message)
+
+    def flush(self):
+        pass
```

Comment:
This should be in a separate class.

Reply:
Ok cool, would also be good if we did this more cleanly. This is fully just a Claude implementation.